lib_xua/src/core/buffer/ep/ep_buffer.xc

```diff
@@ -544,7 +555,21 @@ void XUA_Buffer_Ep(register chanend c_aud_out,
                     t_sofCheck :> timeLastEdge;
                     sofCount = 0;
                     timeNextEdge = timeLastEdge + LOCAL_CLOCK_INCREMENT + LOCAL_CLOCK_MARGIN;
+                    pllUpdate++;
                 }
+#if (XUA_USE_APP_PLL)
+                // Update PLL @ 100Hz
```

Comment:
Not sure why this is 100Hz. Isn't pllUpdate incremented once per second, so the PLL is updated every 10s?

Reply:
For high-speed
- sofCount gets incremented every 125uS
- pllCount incremented every 8 SOFS (for HS)
- pll updated when pllUpdate = 10
- This leads to a period of 10ms (100Hz)